Revised Guthub How-To

joeschofield0 opened the pull request.

BenJamesBen commented on markdown/dev/howtos/git/editing-content-on-github/en.md:
```diff
@@ -0,0 +1,58 @@
+<Note> 
+Clicking **Preview** will show what your changes will look like.
+![Edit and preview buttons](edit-preview.png)
+</Note>
```
The Markdown in this \<Note> section didn't get converted. I am not sure why. If you "View file", you will see that 
1. "\*\*Preview**" remains, and 
2. The edit-preview.png image did not appear.

(Do there need to be blank lines after \<Note> and before \</Note>? I don't know how this aspect of Markdown works.)

joeschofield0 replied:
Thanks.  I used the syntax from the Markdown guide, and [Note ](https://freesewing.dev/guides/markdown/custom-components#note) appears under "Custom components".  Hopefully it will render OK on the site.  I've made a note to confirm this following the pull request.

BenJamesBen commented on markdown/dev/howtos/git/editing-content-on-github/en.md:
```diff
@@ -0,0 +1,58 @@
+
+Congratulations - you have just submitted your first edits to the site maintainers!
+
+You can keep track of the progress of your changes in GitHub or in the [github-updates channel](discord-github-updates.PNG) on our Discord server.
```
I think you are missing the "!" exclamation mark before "[github-updates channel]". The image doesn't appear.

joeschofield0 replied:
Fixed.  Thank you for these.

BenJamesBen reviewed (changes requested):
This PR adds 10 PNG image files. However, the document uses only 7 images. Is your document missing any images? Or, are there any images that you decided not to use and should be omitted from this PR?

joeschofield0 commented:
@BenJamesBen Thanks for your feedback.  I've addressed the points you made (apart from the Note section - this is custom code).  I've committed the en.md edits, and the deletion of redundant image files.  Pull request submitted from joeschofield0-revised-github-how-to --> base:develop

joostdecock commented:
@all-contributors please add @joeschofield0 for docs

joostdecock commented:
Oops, thanks for reminding my bot :)
